Week2/homework/index.js

```diff
@@ -0,0 +1,170 @@
+const pro = require('prompt');
+const mysql = require('mysql');
+const con = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+})
+
+
+
+const searchCapital = `select name 
+from city 
+where id 
+in (select capital from country where name = ?);`
+
+const searchRegion = `select distinct language 
+from countrylanguage 
+where countrycode 
+in(select code from country where region = ?);`
+
+const searchNumOfCities = `select distinct name 
+from city 
+where countrycode 
+in (select countrycode from countrylanguage where language = ?);`
+
+const searchOfficialLanguage = `select name 
+from country 
+where region = ? 
+and code in(select countrycode from countrylanguage where language = ? and isofficial = 't');`;
+
+const getContinentList = `select distinct continent 
```

Comment:
Ok It works using too queries, but you can do it in once :) Think about the group by statement.

Reply:
Ok, I will make it simple.

Reply:
Thanks, during changing my code I have learned a lot.